.github/workflows/ci-sglang-benchmark.yml

```diff
@@ -0,0 +1,88 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: SGLang Llama Benchmarking Tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Weekdays at 4:00 AM UTC = 9:00 PM PST.
+    - cron: "0 4 * * 1-5"
```

Comment:
Since this is only executed on a workflow dispatch or when scheduled, have you tested it? Might be worth to add `  pull_request:` temporarily just for testing.

Reply:
That is a great point. Will do that